bezier_application/src/bezier_application.cpp

```diff
@@ -18,12 +18,12 @@
 /**
  * @file bezier_application.hpp
  * @brief This is an application using bezier_library.
- * @author Francois Lasson - Institut Maupertuis (France)
+ * @author Francois Lasson, Kévin Bolloré - Institut Maupertuis (France)
  * @date Project started in February 2015
  */
 /**@mainpage This application is a test of bezier_library.
  * It has been developed for a Fanuc M10iA Robot under Ubuntu 14.04 with PCL 1.8.0
- * VTK 6.3 (with patch) and ROS Indigo.
+ * VTK 7.0 (with patch) and ROS Indigo.
```

Comment:
Remove this (+ Ubuntu + PCL) and point to the official [README.md](https://github.com/ros-industrial-consortium/bezier/blob/indigo-devel/README.md).
Let's only have 1 source of information, it's easier to maintain.